econml/dml.py

```diff
@@ -333,30 +381,45 @@ class DMLCateEstimator(_DMLCateEstimatorBase):
         If None, the random number generator is the RandomState instance used
         by `np.random`.
 
-    inference: string, inference method, or None
+    inference: string, `Inference` instance, or None
         Method for performing inference.  This estimator supports 'bootstrap'
-        (or an instance of `BootstrapOptions`)
+        (or an instance of `BootstrapInference`) and 'statsmodels' (or an instance of 'StatsModelsInference`)
     """
 
     def __init__(self,
-                 model_y, model_t, model_final=LinearRegression(fit_intercept=False),
+                 model_y=LassoCV(), model_t=LassoCV(),
                  featurizer=PolynomialFeatures(degree=1, include_bias=True),
                  discrete_treatment=False,
                  n_splits=2,
-                 random_state=None,
-                 inference=None):
+                 random_state=None):
         super().__init__(model_y=model_y,
                          model_t=model_t,
-                         model_final=model_final,
+                         model_final=StatsModelsWrapper(),
                          featurizer=featurizer,
-                         sparseLinear=False,
+                         sparseLinear=True,
```

Comment:
There is something counter intuitive and legacy-driven about the sparseLinear name of this parameter. Also here we allow the user to specify arbitrary first stage models but we do not allow them to change the sparseLinear flag. This seems weird as the user might specify a randomforest first stage and then we overkill it by augmenting the features. 

I would propose the following two changes if possible:
1) change the name of the option. Maybe sth like: `linear_first_stages=True`
2) Allow the user to alter the flag even in the LinearDMLCateEstimator and SparseLinearDMLCateEstimator and only have the default be True.

Reply:
Sure, that sounds good.